docs/overview/event.rst

```diff
@@ -188,11 +190,11 @@ Display events
 Custom events
 -------------
 
-Any custom event codes can be registered by
+Any number of custom event codes can be registered by
 ``uint32_t MY_EVENT_1 =`` :cpp:func:`lv_event_register_id`
 
-They can be sent to any object with
-:cpp:expr:`lv_event_send(obj, MY_EVENT_1, &some_data)`
+They can be sent to any objects with
```

Comment:
Hi, Gábor.  @kisvegabor 
Singular "object" was actually correct here, since the "count" of "any" is 1.  So "They can be sent to any object with" is both correct and clear.  Would you like me to pick it up in the task I am working on?

Reply:
> Oh, since it's already merged could you fix it n one of you open docs PRs?

Done!  😄

Reply:
Thanks!